src/cluster/hookfunctions.go

```diff
@@ -93,7 +93,8 @@ func (p *Priority) GetPriority() int {
 
 // BasePostFunction describe a default posthook function
 type BasePostFunction struct {
-	f func(CommonCluster) error
+	f      func(CommonCluster, pkgCluster.PostHookConfig) error
```

Comment:
Instead of this change I would rather add the configuration to the global config (in `internal/global`) and use it in posthooks where necessary.

Posthooks will be decomissioned eventually, so I would keep changes here minimal.